src/test/java/seedu/address/model/person/NameTest.java

```diff
@@ -21,21 +21,36 @@ public void constructor_invalidName_throwsIllegalArgumentException() {
 
     @Test
     public void isValidName() {
-        // null name
-        assertThrows(NullPointerException.class, () -> Name.isValidName(null));
-
-        // invalid name
-        assertFalse(Name.isValidName("")); // empty string
-        assertFalse(Name.isValidName(" ")); // spaces only
-        assertFalse(Name.isValidName("^")); // only non-alphanumeric characters
-        assertFalse(Name.isValidName("peter*")); // contains non-alphanumeric characters
-
-        // valid name
-        assertTrue(Name.isValidName("peter jack")); // alphabets only
-        assertTrue(Name.isValidName("12345")); // numbers only
-        assertTrue(Name.isValidName("peter the 2nd")); // alphanumeric characters
-        assertTrue(Name.isValidName("Capital Tan")); // with capital letters
-        assertTrue(Name.isValidName("David Roger Jackson Ray Jr 2nd")); // long names
+        // Valid names
+        assertTrue(Name.isValidName("John")); // Single word name
+        assertTrue(Name.isValidName("John Doe")); // Standard first and last name
+        //        assertTrue(Name.isValidName("Ana María")); // Name with accented characters
+        //        assertTrue(Name.isValidName("O'Conner")); // Name with apostrophe
+        //        assertTrue(Name.isValidName("D'Artagnan")); // Name with apostrophe
```

Comment:
Are you going to keep these tests? If not, I think you can delete them

Reply:
I previously drafted a more sophisticated regex but I didn't quite understand it haha